tests/cc/test_cfg_none_simple.json

```diff
@@ -0,0 +1,26 @@
+{
+   "eqasm_compiler" : "none",
+
+   "hardware_settings": {
+	 "qubit_number": 17,
+	 "cycle_time" : 20,
+	 "mw_mw_buffer": 0,
```

Comment:
mw_mw_buffer and all remaining ones until } are not used anymore, can be deleted, as far as I know